prac_06/guitar.py

```diff
@@ -0,0 +1,28 @@
+"""Class defining a guitar."""
+
+CURRENT_YEAR = 2021     # Improvement: Use computer's internal clock
+VINTAGE_AGE = 50
+
+
+class Guitar:
+    """Represents a Guitar object."""
+
+    def __init__(self, name="", year=0, cost=0):
+        """Initialise a Guitar instance."""
+        self.name = name
+        self.year = int(year)
+        self.cost = float(cost)
```

Comment:
You could simplify this line by writing it as self.cost = cost

Reply:
This is same as reflection in programming_language.py. Refer to that comment.